public/app/features/alerting/unified/components/receivers/onCall/onCall.ts

```diff
@@ -0,0 +1,32 @@
+import { AppPluginMeta } from '@grafana/data';
+import { useGetSingle } from 'app/features/plugins/admin/state/hooks';
+import { getPluginSettings } from 'app/features/plugins/pluginSettings';
+
+import { Receiver } from '../../../../../../plugins/datasource/alertmanager/types';
+import { CatalogPlugin } from '../../../../../plugins/admin/types';
+
+// can we assume this value will be always this one?
+// ideally => BE returns this value
+export const ONCALL_PLUGIN_ID = 'grafana-oncall-app';
+
+export const useGetOnCallIsInstalledAndEnabled = () => {
+  const plugin: CatalogPlugin | undefined = useGetSingle(ONCALL_PLUGIN_ID);
+  return plugin?.isInstalled && !plugin?.isDisabled && plugin?.isPublished && plugin?.type === 'app'; // fetches the plugin settings for this Grafana instance
+};
+
+export const isInOnCallIntegrations = (url: string, integrationsUrls: string[]) => {
+  return integrationsUrls.findIndex((integrationUrl) => integrationUrl === url) !== -1;
+};
+
+export const isOnCallReceiver = (receiver: Receiver, integrationsUrls: string[]) => {
+  if (!receiver.grafana_managed_receiver_configs) {
+    return false;
+  } else {
```

Comment:
You don't really need the `else` statement if you're doing an early return. I find it clearer without it as it removes a nesting level for the remaining code.